core/src/main/java/com/linecorp/armeria/server/file/HttpFile.java

```diff
@@ -200,11 +200,12 @@ static HttpFileBuilder builder(URL url) {
 
             return builder(f.toPath());
         } else if ("jar".equals(url.getProtocol()) && url.getPath().startsWith("file:") ||
-                   "jrt".equals(url.getProtocol())) {
+                   "jrt".equals(url.getProtocol()) ||
+                   "bundle".equals(url.getProtocol())) {
             return new ClassPathHttpFileBuilder(url);
```

Comment:
So this `ClassPathHttpFileBuilder` supports the bundle scheme correctly, right?

Reply:
yes, as it was working before the check introduced here
The actual handling of the scheme is done by OSGi